.github/workflows/deploy.yml

```
@@ -16,7 +16,9 @@ jobs:
       - name: Install Dependencies
         run: yarn install
       - name: Build Docs
-        run: BASE='/guidelines/' yarn docs:build
+        run: yarn docs:build
+      - name: Add CNAME file
+        run: echo 'guidelines.elbgoods.de' > ./dist/CNAME
```

Sorry, der Pfad war falsch kopiert. Der build Ordner ist hier `docs/.vuepress/dist/`
```suggestion
        run: echo 'guidelines.elbgoods.de' > ./docs/.vuepress/dist/CNAME
```